research/gnn-survey/layers.py

```diff
@@ -18,16 +18,18 @@
 class GraphConvLayer(tf.keras.layers.Layer):
   """Single graph convolution layer."""
 
-  def __init__(self, output_dim, bias, **kwargs):
+  def __init__(self, output_dim, sparse, bias, **kwargs):
     """Initializes the GraphConvLayer.
 
     Args:
       output_dim: (int) Output dimension of gcn layer
+      sparse: (bool) sparse: Whether features are sparse
```

Comment:
Should this have a default instead of being required?

Reply:
I think that it should be default as True but I'm not sure that if someone wants to use `False` settings.
Can I just add the default value in the init function but do not change the other part of it? (i.e. models.py)